common/beerocks/bwl/dummy/base_wlan_hal_dummy.h

```diff
@@ -51,6 +51,19 @@ class base_wlan_hal_dummy : public virtual base_wlan_hal,
     virtual bool process_nl_events() override;
     virtual std::string get_radio_mac() override;
 
+    /**
+     * @brief Gets channel utilization.
+     *
+     * @see base_wlan_hal::get_channel_utilization
```

Comment:
:+1: 

As @RanRegev noted in #1309 we should have documentation of how to write doxygen, and this should be part of it.